### ISSUE TITLE
AwesomeOpenSource

### ISSUE THREAD

**Issue by jeffreyjackson** [cjbarber/ToolsOfTheTrade#255]
We are creating a github org for all awesome open source repos. I am working with a few individuals who are also managing their own awesome repos, and I think to gain more traction we need to work together. What are your thoughts about collectively moving your awesome repos to https://github.com/AwesomeOpenSource

The idea is that visibility for these repos would be increased due to more traffic to the org. What are your thoughts? Let me know if there is anything you'd like to discuss!

**Comment by perfaram** [comment]
I don't see the point

**Comment by jeffreyjackson** [comment]
We are working on creating an open organization to further improve the awesomeness :grimacing:

1. By having everything under 1 organization visibility will be increased for all awesome repos -> more stars -> round we go.
2. Ownership is shared so there is less risk of the repo falling off unsupported.
3. The most popular awesome list isn't currently being updated as often as we'd like.  Having it under an organization should help assist in keeping it current.

tl;dr
more stars, more people, more support

the original admin will continue to have admin rights so they'll have the same level of control as always.

**Comment by perfaram** [comment]
Okay, then... seems good to me !

**Comment by jeffreyjackson** [comment]
@cjbarber what are your thoughts?

**Comment by perfaram** [comment]
It is sad, that few people will actually go to the author's page to see his other projects and/or follow him.
Apart from this, it seems positive (more repos will get starred), and the 2nd point is a really good thing also.

**Comment by perfaram** [comment]
And sorry for interfering :worried:

**Comment by jeffreyjackson** [comment]
I agree @perfaram - thank you for helping.  Any ideas that you have to help this off the ground please let me know!
